app/services/branch_title_generator.rb

```diff
@@ -0,0 +1,16 @@
+module BranchTitleGenerator
+  def title
+    titles = branches.map(&:title)
+    possible_title = branches.size + 1
+    name = "Branching #{possible_title}"
```

Comment:
current design says `Branching node . . .`

Reply:
Yes, I was waiting after standup to change and move to a locale. That's also done! 🚀 🚀 🚀